hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/net/TestNetUtils.java

```diff
@@ -39,6 +39,7 @@
 import java.util.concurrent.TimeUnit;
 
 import org.apache.commons.lang3.StringUtils;
+import org.apache.ftpserver.command.impl.PORT;
```

Comment:
Why do we need this?

Reply:
Thanks for review, fixed as you mentioned